packages/schematics/angular/utility/latest-versions.ts

```diff
@@ -16,4 +16,8 @@ export const latestVersions = {
   // The versions below must be manually updated when making a new devkit release.
   DevkitBuildAngular: '~0.800.0-beta.13',
   DevkitBuildNgPackagr: '~0.800.0-beta.13',
+  DevkitBuildWebpack: '~0.800.0-beta.13',
```

Comment:
The beta specifier shouldn’t be needed anymore.

Reply:
Oops, PTAL